### ISSUE TITLE
Rails 7: [NameError] uninitialized constant ActiveSupport::XmlMini::IsolatedExecutionState

### ISSUE THREAD

**Issue by hasan-ally** [iftheshoefritz/solargraph-rails#25]
When used with rails 7, get below error

```
[INFO] Preparing library for /Users/hasan/projects/aksharainnovations/campuspass
[INFO] Indexing workspace files in /Users/hasan/projects/aksharainnovations/campuspass
[WARN] Error processing request: [NameError] uninitialized constant ActiveSupport::XmlMini::IsolatedExecutionState

        IsolatedExecutionState[:xml_mini_backend]
        ^^^^^^^^^^^^^^^^^^^^^^
[WARN] /Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/xml_mini.rb:185:in `current_thread_backend'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/xml_mini.rb:103:in `backend='
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/xml_mini.rb:202:in `<module:ActiveSupport>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/xml_mini.rb:11:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/array/conversions.rb:3:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/array/conversions.rb:3:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/duration.rb:3:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/duration.rb:3:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/time/calculations.rb:3:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/time/calculations.rb:3:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/string/conversions.rb:4:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/string/conversions.rb:4:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/string.rb:3:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/bundler/gems/rails-f466a4f09fdb/activesupport/lib/active_support/core_ext/string.rb:3:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-rails-0.2.2.pre.4/lib/solargraph/rails/pin_creator.rb:3:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-rails-0.2.2.pre.4/lib/solargraph/rails/pin_creator.rb:3:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-rails-0.2.2.pre.4/lib/solargraph-rails.rb:5:in `require_relative'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-rails-0.2.2.pre.4/lib/solargraph-rails.rb:5:in `<top (required)>'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/workspace.rb:207:in `require'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/workspace.rb:207:in `block in require_plugins'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/workspace.rb:205:in `each'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/workspace.rb:205:in `require_plugins'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/workspace.rb:34:in `initialize'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/library.rb:410:in `new'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/library.rb:410:in `load'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host.rb:291:in `prepare'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host.rb:309:in `block in prepare_folders'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host.rb:308:in `each'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host.rb:308:in `prepare_folders'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/message/initialize.rb:11:in `process'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host.rb:112:in `receive'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host/message_worker.rb:53:in `tick'
/Users/hasan/.asdf/installs/ruby/3.1.0-preview1/lib/ruby/gems/3.1.0/gems/solargraph-0.44.2/lib/solargraph/language_server/host/message_worker.rb:44:in `block in start'
[INFO] Sending response to initialize
[DEBUG] Content-Length: 227

{"jsonrpc":"2.0","id":0,"error":{"code":-32603,"message":"[NameError] uninitialized constant ActiveSupport::XmlMini::IsolatedExecutionState\n\n        IsolatedExecutionState[:xml_mini_backend]\n        ^^^^^^^^^^^^^^^^^^^^^^"}}
[INFO] Server received shutdown
[DEBUG] {"jsonrpc"=>"2.0", "id"=>1, "method"=>"shutdown"}
[INFO] Sending response to shutdown
[DEBUG] Content-Length: 38

{"jsonrpc":"2.0","id":1,"result":null}
```

**Comment by iftheshoefritz** [comment]
This should be fixed with v0.3.0. @hasan-ally would you mind checking and closing this if so?

**Comment by hasan-ally** [comment]
Thanks @iftheshoefritz 
New version works
